src/contracts/examples/redstone-oev/RedstoneDAppControl.sol

```diff
@@ -0,0 +1,108 @@
+//SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.25;
+
+import { DAppControl } from "src/contracts/dapp/DAppControl.sol";
+import { CallConfig } from "src/contracts/types/ConfigTypes.sol";
+import "src/contracts/types/UserOperation.sol";
+import "src/contracts/types/SolverOperation.sol";
+import "./RedstoneAdapterAtlasWrapper.sol";
+import { SafeTransferLib } from "solady/utils/SafeTransferLib.sol";
+
+contract RedstoneDAppControl is DAppControl {
+    error InvalidBaseFeed();
+    error InvalidRedstoneAdapter();
+    error FailedToAllocateOEV();
+    error OnlyGovernance();
+
+    uint256 private bundlerOEVPercent = 5;
+
+    event NewRedstoneAtlasAdapterCreated(address indexed wrapper, address indexed owner, address indexed baseFeed);
+
+    mapping(address redstoneAtlasAdapter => bool isAdapter) public isRedstoneAdapter;
+
+    constructor(address _atlas)
+        DAppControl(
+            _atlas,
+            msg.sender,
+            CallConfig({
+                userNoncesSequential: false,
+                dappNoncesSequential: false,
+                requirePreOps: false,
+                trackPreOpsReturnData: false,
+                trackUserReturnData: true,
+                delegateUser: false,
+                requirePreSolver: false,
+                requirePostSolver: false,
+                requirePostOps: false,
+                zeroSolvers: false,
+                reuseUserOp: false,
+                userAuctioneer: true,
+                solverAuctioneer: false,
+                unknownAuctioneer: true,
```

Comment:
unknownAuctioneer is quite a risky setting to turn on I think. Will we want anyone to be able to sign the dAppOp?